### ISSUE TITLE
Annoying menu settings tooltips prevents clicking upper menu entries. Tooltips don't dissapear when moving the cursor away.

### ISSUE THREAD

**Issue by yevon** [microsoft/terminal#14184]
### Windows Terminal version

1.15.2713.0

### Windows build number

22621.525

### Steps to reproduce

1. Open Windows Terminal settings. 
2. Hover the cursor over a setting, and a tooltip will appear near the cursor over the upper menu setting.
3. If you move the cursor slowly over the tooltip, the tooltip doesn't disappear and you can't click de upper menu entry.

![image](https://user-images.githubusercontent.com/121157/195131189-893c4085-162c-4058-bbf0-53b24244ccd9.png)

This is quite annoying, tooltips should dissapear immediately after moving the cursor in any direction.

### Actual Behavior

Tooltips don't dissapear when moving the cursor and prevents from clicking settings menu entries.

**Comment by yevon** [comment]
I would say that those tooltips are completely useless, I haven't seen them in any other UWP app, like powertoys or windows settings app.

**Comment by DHowett** [comment]
Thanks for the report! There's a couple things at play here.

> completely useless

They are useful when you have long profile names or are in collapsed navigation mode:

<img width="292" alt="image" src="https://user-images.githubusercontent.com/189190/195397635-186b61f2-77ef-4e5e-83d9-f00619d02258.png">

<img width="112" alt="image" src="https://user-images.githubusercontent.com/189190/195397664-399a36a5-7020-4bb3-ad66-c80b9d673c80.png">

FWIW, I didn't think that we should put tooltips into the navigation tray in Settings for the same reason (it offers no additional information,) but I was convinced by the argument that it assists in cases where the text legitimately doesn't fit.

> doesn't disappear

Interesting; this appears to be a conscious design choide for the modern UI framework. Take the taskbar clock for example: if your mouse enters the tooltip above the clock, it stays open and you can't click whatever's behind it.

<img width="155" alt="image" src="https://user-images.githubusercontent.com/189190/195398020-4c80fcab-45c1-4f4e-b8b0-e29c3e163f09.png">

Visual Studio uses WPF (primarily) and has original behavior (the tooltip disappears if you move out of range of the original item).

This one is bordering on "by design" given that it's a systemwide choice to make tooltips block interaction... 🤔

**Comment by yevon** [comment]
You are right this seems made by design and I don't really understand it, as you never interact with tooltips you shouldn't be able to press them ever. Tooltips should dissappear once the cursor is not over the element that you are trying to press. I haven't seen this behaviour outside microsoft apps, see for example the spotify app. If you place the cursor over the play button a tooltip appears, but immediately dissapears when you place your cursor outside of it which is the standard behaviour of tooltips.

In the taskbar example is also very annoying, if you have an application behind the tooltip, you can't press it.

Is this UWP related? Where should I post this then?

Standard behaviour of tooltips in all apps I know of is:

- Don't show a tooltip if the text is fully visible.
- Tooltips should remain as far as you keep the cursor hover in the element.


Maybe here?

https://github.com/microsoft/WindowsAppSDK
 or here?

https://github.com/microsoft/microsoft-ui-xaml

**Comment by DHowett** [comment]
Ruling from the triage meeting: we should get rid of the redundant tooltips. We should _also_ pursue the issue on the XAML side (thank you for filing it!)
